document/src/vespa/document/fieldvalue/tensorfieldvalue.cpp

```diff
@@ -20,6 +22,13 @@ namespace {
 
 TensorDataType emptyTensorDataType;
 
+vespalib::string makeWrongTensorTypeMsg(const ValueType &fieldTensorType, const ValueType &tensorType)
+{
+    return vespalib::make_string("Field tensor type is '%s' but tensor type is '%s'",
```

Comment:
Consider adding 'other' to make message a little clearer: '... but other tensor type ...'